crates/rolldown/tests/rolldown/warnings/none-ascii-content/artifacts.snap

```diff
@@ -8,7 +8,7 @@ snapshot_kind: text
 
 ```text
 [UNRESOLVED_IMPORT] Warning: Could not resolve 'test' in main.js
-   ╭─[ main.js:2:14 ]
+   ╭─[ main.js:2:18 ]
```

Comment:
This is expected, since diagnostic is one based.

![image](https://github.com/user-attachments/assets/bbfedd78-4e5b-40d1-949a-29c28538aba3)